packages/app-project/src/components/ProjectHeader/components/DropdownNav/DropdownNav.stories.js

```diff
@@ -0,0 +1,126 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Grommet } from 'grommet'
+import { Provider } from 'mobx-react'
+import { applySnapshot } from 'mobx-state-tree'
+
+import DropdownNav from './DropdownNav.js'
+import initStore from '@stores'
+
+function ThemedStory(Story) {
+  return (
+    <Grommet theme={zooTheme}>
```

Comment:
```suggestion
    <Grommet theme={zooTheme} background='dark-3'>
```

A darker background makes the light text easier to ready in storybook.